### PR TITLE
[ANCHOR-294] test: create SEP31 e2e tests

### PR DESCRIPTION
**Pull request by paulormnas**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [x] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Add Kotlin e2e tests for SEP31 flow

### Why

To deprecate Python e2e tests

### Known limitations

N/A

### CODE REVIEW

**Comment by lijamie98** [comment]
Thanks for making this PR to deprecate the SEP-31 python scripts.

1. Please make the tests to pass. 
2. Please remove the sep-31 python scripts.

File: integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt [hunk 1]
```diff
@@ -118,7 +118,7 @@ class Sep24End2EndTest(
     // Submit transfer transaction
     val txn = (anchor.getTransaction(withdrawal.id, token) as WithdrawalTransaction)
     val transfer =
-      wallet.stellar().transaction(txn.from).transferWithdrawalTransaction(txn, asset).build()
+      wallet.stellar().transaction(keypair).transferWithdrawalTransaction(txn, asset).build()
```
Comment (lijamie98) [inline integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt:121]: Is this intentional?

File: integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep31End2EndTests.kt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  private fun createTransaction(
+    transactionPayload: Sep31PostTransactionRequest,
+    sep31Client: Sep31Client
+  ): Sep31PostTransactionResponse {
+    return sep31Client.postTransaction(transactionPayload)
+  }
+
+  private suspend fun waitStatus(id: String, expectedStatus: String, sep31Client: Sep31Client) {
```
Comment (lijamie98) [inline integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep31End2EndTests.kt:120]: Sep24End2EndTests has similar function. Please generalize and reuse.

**Review by lijamie98 — CHANGES_REQUESTED** [review]

File: integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt (hunk 1 above)
Comment (paulormnas) [reply integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/Sep24End2EndTests.kt:116]: Yes, it is necessary due to a wallet version upgrade

**Pull request closed without merging**